src/topology/stone_cech.lean

```diff
@@ -230,6 +230,10 @@ def stone_cech_unit (x : α) : stone_cech α := ⟦pure x⟧
 lemma dense_range_stone_cech_unit : dense_range (stone_cech_unit : α → stone_cech α) :=
 dense_range_pure.quotient
 
+lemma stone_cech_unit_dense :
+  closure (set.range (stone_cech_unit : α → stone_cech α)) = set.univ :=
+dense.closure_eq dense_range_stone_cech_unit
+
```

Comment:
```suggestion
```
The proof could be `dense_range_stone_cech_unit.closure_range` which already suggests this lemma isn't so useful. But there is much worse. Its unique use is on line 72-73 of `extremally_disconnected`:
```lean
  have H : dense_range (stone_cech_unit : X → stone_cech X),
  { rw dense_range_iff_closure_range, exact stone_cech_unit_dense },
```
which could be 
```lean
  have H : dense_range (stone_cech_unit : X → stone_cech X),
  { exact dense_range_stone_cech_unit },
```
so the current proof first transform the goal into something more complicated and then applies a useless lemma.